source/adapters/cuda/event.hpp

```diff
@@ -10,7 +10,7 @@
 #pragma once
 
 #include <cuda.h>
-#include <ur/ur.hpp>
+#include <ur_api.h>
```

Comment:
There is a distinction between the `#include <ur_api.h>` (*h-char-sequence*) and `#include "ur_api.h" `(*q-char-sequence*). Though the c99 standard (and by reference c++11 - c++20) leave latitude for interpretation of these two forms, it's standard practise to use the "q" form for internal includes and the "h" form for system includes (see e.g. llvm).
 
Since ur_api.h is part of the present project, I believe all instances should the quoted form. Such a general cleanup certainly belongs in a separate patch, but for all changes introduced changes here, I think it makes sense to use q-char uniformly